### PR TITLE
fix: patch to build with crystal 1.9.1

### LINKED ISSUE #1329: amber 1.3.2 regression test failure against crystal 1.9.1
### Description

While building crystal 1.9.1, it was found that amber 1.3.2 test failed

```
  Postinstall of ameba: make bin && make run_file
  Failed postinstall of ameba on make bin && make run_file:
  /usr/local/bin/shards build 
  Dependencies are satisfied
  Building: ameba
  Error target ameba failed to compile:
  Showing last frame. Use --error-trace for full trace.
  
  There was a problem expanding macro 'macro_4732366672'
  
  Code in /usr/local/Cellar/crystal/1.9.1/share/crystal/src/yaml/serialization.cr:188:7
  
   188 | {% begin %}
         ^
  Called macro defined in /usr/local/Cellar/crystal/1.9.1/share/crystal/src/yaml/serialization.cr:188:7
  
   188 | {% begin %}
  
  Which expanded to:
  
     75 |                   
     76 | 
   > 77 |                   __temp_722 =
                            ^
  Error: type must be Ameba::Severity, not (Ameba::Severity | Nil)
  
  make: *** [bin/ameba] Error 1
```

### Steps to Reproduce

1. [First Step]
2. [Second Step]
3. [and so on...]

**Expected behavior:** [What you expect to happen]

pass

**Actual behavior:** [What actually happens]

failed

**Reproduces how often:** [What percentage of the time does it reproduce?]

### Additional Information

relates to https://github.com/Homebrew/homebrew-core/pull/136821

### PR DESCRIPTION
**Pull request by chenrui333**
closes #1329
relates to https://github.com/Homebrew/homebrew-core/pull/136915

cc @crimson-knight

### CODE REVIEW

**Review by crimson-knight — APPROVED** [review]
Looks good, thank you!

I'll merge this later and make a PR on Homebrew core with an updated release for Amber.

**Comment by chenrui333** [comment]
That would be great. I will close my associated homebrew one then. 💯

**Comment by chenrui333** [comment]
@crimson-knight looks like there is a new crystal 1.9.2 release now, can you merge and make a release for amber? Thanks!!

**Comment by chenrui333** [comment]
@crimson-knight gentle ping 🙏

File: shard.yml [hunk 1]
```diff
@@ -81,4 +81,4 @@ dependencies:
 development_dependencies:
   ameba:
     github: crystal-ameba/ameba
-    version: ~> 1.3.0
+    branch: master
```
Comment (Blacksmoke16) [inline shard.yml:84]: At this point, probably be better to upgrade to https://github.com/crystal-ameba/ameba/releases/tag/v1.5.0 instead of master.

**Comment by crimson-knight** [comment]
@chenrui333 thank you for the follow up, I should be able to get this fixed today. There's a couple of things I want to include in the new release. I was just delayed because I had a family member admitted to the hospital last week. It took some time but they will be ok now.

**Comment by crimson-knight** [comment]
I resolved this with another commit. I'll submit what is necessary for Homebrew shortly

**Pull request closed without merging**